project/AkkaDisciplinePlugin.scala

```diff
@@ -21,91 +21,71 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
   override def requires: Plugins = JvmPlugin && ScalafixPlugin
   override lazy val projectSettings = disciplineSettings
 
-  val fatalWarningsFor = Set(
-    "akka-discovery",
-    "akka-distributed-data",
-    "akka-coordination",
-    "akka-protobuf"
-  )
+  val fatalWarningsFor =
+    Set("akka-discovery", "akka-distributed-data", "akka-coordination", "akka-protobuf", "akka-cluster-tools")
```

Comment:
I liked them on separate lines, but I guess this is the autoformatter kicking in... ok :(

Reply:
The funky formatter is the culprit. (I agree on the LF:y one being nicer)

Reply:
Resistance is futile